api/application/client.go

```diff
@@ -580,6 +580,41 @@ func (c *Client) DestroyConsumedApplication(saasNames ...string) ([]params.Error
 	return allResults, nil
 }
 
+// ScaleApplicationParams contains parameters for the ScaleApplication API method.
+type ScaleApplicationParams struct {
```

Comment:
That's new :) I like the idea of having a struct for params. we have not done that though for api calls.